### PR TITLE
Add support for Cypress v10

### PR DESCRIPTION
**Pull request by pranavj1001**

### CODE REVIEW

**Review by roshan04 — APPROVED** [review]

File: bin/helpers/constants.js [hunk 1]
```diff
@@ -120,15 +120,19 @@ const validationMessages = {
   EMPTY_RUN_SETTINGS: "Empty run settings",
   EMPTY_CYPRESS_PROJ_DIR:
     "cypress_proj_dir is not set in run_settings. See https://www.browserstack.com/docs/automate/cypress/sample-tutorial to learn more.",
+  EMPTY_CYPRESS_CONFIG_FILE: 
+    "cypress_config_file is not set in run_settings. See https://www.browserstack.com/docs/automate/cypress/sample-tutorial to learn more.",
```
Comment (sreenivas-k) [inline bin/helpers/constants.js:124]: Can we give this docs link: https://www.browserstack.com/docs/automate/cypress/specs-to-run

@pranavj1001

Comment (pranavj1001) [reply bin/helpers/constants.js]: Thanks for the comment, however, I've changed the link to https://www.browserstack.com/docs/automate/cypress/configuration-file

**Review by roshan04 — APPROVED** [review]

**Comment by PraveenAsokan** [comment]
@pranavj1001  When this will be released? Any idea?

**Comment by rrsai** [comment]
Doc link comments on this appear to be updated.  I wonder if this will allow for support of cypress 10's move to the cypress.config.js file?  Currently cannot use the browserstack automation because I'm using the latest version of cypress and the config files appear to be non-compatible.

**Pull request merged**